src/main/resources/application.yml

```diff
@@ -42,14 +42,100 @@ nhais:
     password: ${NHAIS_MONGO_PASSWORD:}
     options: ${NHAIS_MONGO_OPTIONS:}
     autoIndexCreation: ${NHAIS_MONGO_AUTO_INDEX_CREATION:true}
-  mesh:
-    mailboxId: ${NHAIS_MESH_MAILBOX_ID:}
-    mailboxPassword: ${NHAIS_MESH_MAILBOX_PASSWORD:}
-    sharedKey: ${NHAIS_MESH_SHARED_KEY:}
-    host: ${NHAIS_MESH_HOST:}
-    endpointCert: ${NHAIS_MESH_ENDPOINT_CERT:}
-    endpointPrivateKey: ${NHAIS_MESH_ENDPOINT_PRIVATE_KEY:}
-    cypherToMailbox: ${NHAIS_MESH_CYPHER_TO_MAILBOX:}
+  mesh: # sensible defaults for local development using fake-mesh
+    mailboxId: ${NHAIS_MESH_MAILBOX_ID:gp_mailbox}
+    mailboxPassword: ${NHAIS_MESH_MAILBOX_PASSWORD:password}
+    sharedKey: ${NHAIS_MESH_SHARED_KEY:SharedKey}
+    host: ${NHAIS_MESH_HOST:https://localhost:8829/messageexchange/} #local fake-mesh default endpoint
+    endpointCert: ${NHAIS_MESH_ENDPOINT_CERT:-----BEGIN CERTIFICATE-----
```

Comment:
Those shouldn't be here?

Reply:
Sensible defaults for development - so you can just run the thing without lots of tedious setup